explore/src/main/scala/explore/config/sequence/GeneratedSequenceViewer.scala

```diff
@@ -38,21 +42,26 @@ object GeneratedSequenceViewer:
     ScalaFnComponent
       .withHooks[Props]
       .useContext(AppContext.ctx)
-      .useStreamResourceOnMountBy { (props, ctx) =>
+      .useStreamResourceOnMountBy: (props, ctx) =>
         import ctx.given
 
+        val targetChanges = props.targetIds.traverse: targetId =>
+          TargetQueriesGQL.TargetEditSubscription
+            .subscribe[IO](targetId)
+
         SequenceQuery[IO]
           .query(props.obsId)
           .map(_.observation.map(_.execution.config))
           .attemptPot
           .resetOnResourceSignals(
-            ObsQueriesGQL.ObservationEditSubscription.subscribe[IO](props.obsId)
+            for {
+              o <- ObsQueriesGQL.ObservationEditSubscription.subscribe[IO](props.obsId)
+              t <- targetChanges
```

Comment:
It seems that we subscribe to all the changes in the observation and all the targets and rerequest the sequence in those cases.

Can we limit the sequence requests to only when relevant fields change?

Reply:
I was thinking something along the lines of having the relevant fields in the the stream of changes and analyzing if any of them changes before emitting a signal in the final stream.

This may be overkill though. I think the ODB caches the sequence, which wouldn't make it that costly to overfetch it.

Reply:
right not every change will actually change the sequence, it would basically be the target brightness and profile. Not sure how doable is that with the front end only